htsohm/htsohm.py

```diff
@@ -474,7 +501,7 @@ def calculate_mutation_strength(run_id, generation, mutation_strength_bin):
             session.commit()
         except (FlushError, IntegrityError) as e:
             print("Somebody beat us to saving a row with this generation. That's ok!")
-            session.rollback()
+            Session.rollback()
```

Comment:
Why did you change this? I don't think this is right... did you try it in the console?